src/snowflake/cli/api/constants.py

```diff
@@ -33,7 +33,19 @@ def __str__(self):
         return self.sf_name
 
 
-class ObjectType(Enum):
+class BaseObjectType(Enum):
+    def __str__(self):
+        """This makes using this Enum easier in formatted string"""
+        return self.value.cli_name
+
+
+class UseObjectType(BaseObjectType):
+    DATABASE = ObjectNames("database", "database", "databases")
+    ROLE = ObjectNames("role", "role", "roles")
+    WAREHOUSE = ObjectNames("warehouse", "warehouse", "warehouses")
```

Comment:
Sorry, somehow I had missed this. These changes don't make sense to me. If we wanted to group things by account-level, db-level and schema-level, maybe. But "UseObjectType" isn't really a thing IMHO. We can use a union + a type alias to achieve the same thing, and not modify this file at all.

Reply:
reverted the changes to this file and created a separate enum for facade for objects that are "usable".